testing/fpgaconf/test_fpgaconf_c.cpp

```diff
@@ -750,8 +769,8 @@ TEST_P(fpgaconf_c_p, encoding_path) {
  * @test       relative_path
  * @brief      Test: fpgaconf_main
  * @details    When gbs file locates in parent directory and command params<br>
- *             contains path traversal, fpgaconf_main displays file<br>
- *             error and returns non-zero.<br>
+ *             contains path traversal. On success, fpgaconf_main loads bitstream<br>
+ *             and returns zero. Otherwise, it displays file error and returns non-zero.<br>
  */
 TEST_P(fpgaconf_c_p, relative_path) {
```

Comment:
I thought we would expect this one to fail?

Reply:
Previously, it was a false positive on set_userclk. Setup has been modified to load in a valid bitstream so this test succeeds now.

Reply:
Ah, cool.